core/src/test/java/io/questdb/test/cutlass/http/RetryIODispatcherTest.java

```diff
@@ -401,29 +401,6 @@ public void testImportsCreateAsSelectAndDrop() throws Exception {
                 });
     }
 
-    @Test
-    public void testImportsHeaderIsNotFullyReceivedIntoReceiveBuffer() throws Exception {
```

Comment:
Seems to be a valid test case, why is it removed?

Reply:
this is not a valid test, we should (and are able) to parse csv input with buffer size set to 1 byte. The exception was artificial. The header parser uses its own buffer, which is large enough for the test.

After I removed artificial check this test stopped making sense